src/app/dynamic-forms/fields/datetime/datetime.component.scss

```diff
@@ -0,0 +1,34 @@
+:host {
+  .form-control {
+    display: flex;
+    padding: 0 0 0 0.75rem;
+
```

Comment:
No, this is to push the input away from the left wall of the container. The spacing is really tightly-controlled to ensure that the component's size matches the other inputs, since in their cases the input itself has the `.form-control` class